Assets/HoloToolkit/SpatialSound/Scripts/AudioLoFiEffect.cs

```diff
@@ -0,0 +1,222 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// An audio effect that limits the frequency range of a sound to simulate
+    /// being played over various telephony or radio sources.
+    /// </summary>
+    /// <remarks>
+    /// For the best results, also attach an AudioInfluencerManager to the sound
+    /// source. This will ensure that the proper frequencies will be restored
+    /// when audio influencers are used in the scene.
+    /// </remarks>
+    [RequireComponent(typeof(AudioSource))]
+    [RequireComponent(typeof(AudioLowPassFilter))]
+    [RequireComponent(typeof(AudioHighPassFilter))]
+    [DisallowMultipleComponent]
+    public class AudioLoFiEffect : MonoBehaviour
+    {
+        /// <summary>
+        /// The quality level of the simulated audio source (ex: AM radio).
+        /// </summary>
+        [Tooltip("The quality level of the simulated audio source.")]
+        public AudioSourceQuality SourceQuality;
```

Comment:
please use encapsulation with a serialized private field so you can set the quality in the editor.

Reply:
I am able to set the quality in the editor. this also allows it to be changed via code.

Reply:
It's not just about editor access @davidkline-ms , it's more of a stanards thing with Unity.  The recommended standard to use with such fields is to have a Private Field that is serialised (which makes it visible in the editor), then only add the public variant if you intend for the field to be accessed outside of the class.

Reply:
@SimonDarksideJ, are you advocating having BOTH a private and a public field? or are you advocating for a private field and a public set of Get/Set methods? or are you recommending a differently named public property?

For something that was originally intended as being public, it seems that a single public field makes sense.

Reply:
>  are you advocating having BOTH a private and a public field?

No private field with public property. We're trying to ensure encapsulation.